quinn-proto/src/connection/stats.rs

```diff
@@ -152,6 +152,8 @@ pub struct PathStats {
     pub lost_plpmtud_probes: u64,
     /// The number of times a black hole was detected in the path
     pub black_holes_detected: u64,
+    /// Current MTU discovered
+    pub current_mtu: u16,
```

Comment:
I think this could benefit from a terser name, and more detailed doc:
```suggestion
    /// Largest UDP payload size the path currently supports
    pub mtu: u16,
```

Reply:
Updated the doc.

For the field name wouldn't it be better we use the same name as elsewhere in the quinn library, specifically in `MtuDiscovery`:
https://github.com/quinn-rs/quinn/blob/c0b1e281e/quinn-proto/src/connection/mtud.rs#L11
since that is also the source of this value?

Reply:
On second thought I think this is useful disambiguation wrt. the various MTU-related config parameters regardless.